src/cls/rbd/cls_rbd_client.cc

```diff
@@ -594,6 +594,34 @@ namespace librbd {
                                   protection_statuses);
     }
 
+    void snap_timestamp_list_start(librados::ObjectReadOperation *op,
+                                   const std::vector<snapid_t> &ids)
+    {
+      for (vector<snapid_t>::const_iterator it = ids.begin(); it != ids.end(); ++it) {
```

Comment:
I'd rather it's merged into this PR -- things get lost if we "punt" to later and it's a trivial change.